src/systems.rs

```diff
@@ -204,7 +211,13 @@ pub fn process_output(
 
     if let Some(window) = windows.get_primary() {
         if let Some(winit_window) = winit_windows.get_window(window.id()) {
-            winit_window.set_cursor_icon(egui_to_winit_cursor_icon(output.cursor_icon));
+            match egui_to_winit_cursor_icon(output.cursor_icon) {
+                None => winit_window.set_cursor_visible(false),
+                Some(icon) => {
+                    winit_window.set_cursor_visible(true);
```

Comment:
This line causes a dead-lock on my system O_o

Reply:
Weird. The `egui::CursorIcon` got a new variant `None` which isn't present in `winit::window::CursorIcon`.

Maybe setting the cursor to visible or invisible every frame isn't the way to go :sweat_smile:

Reply:
I pushed a commit which only sets the cursor visibility if it is changed. Can you try if that fixes it?

Reply:
Btw, do we need it or do we need to do this in such a way? I tried to look up how Bevy manages window cursors and found this:
https://github.com/bevyengine/bevy/blob/100e516014d803392a81a4e001d3e6ddf5cca414/crates/bevy_winit/src/lib.rs#L115-L118
Which led me to this: https://docs.rs/bevy_window/0.5.0/bevy_window/struct.Window.html#method.set_cursor_visibility

One of my other concerns is that a user might have `WindowDescriptor::cursor_visible` configured to false, while this code may override this. I'm thinking that we should probably only manage cursor icons here and let users control cursor visibility. What do you think?

I guess, if `egui_to_winit_cursor_icon` doesn't return anything, we can either stick to the default cursor or don't change it at all.

Reply:
Done. Now the cursor icon doesnt change if `None` is returned.